docs/developing_recognizers.md

```diff
@@ -45,46 +46,64 @@ See [this documentation](docs/custom_fields.md#via-code) on adding a new recogni
+spaCy provides descent results compared to state-of-the-art NER models, but with much better computational performance.
+spaCy could be trained from scratch, used in combination with pre-trained embeddings, or retrained to detect new entities.
 When building such model, you would need to extend the [EntityRecognizer](presidio-analyzer/presidio_analyzer/entity_recognizer.py) class.
 
 #### Utilize Scikit-learn or similar
 
-Scikit-learn models tend to be fast, but usually have lower accuracy than deep learning methods. However, for well defined problems with well defined features, they can provide very good results. 
-Since deep learning models tend to be complex and slow, we encourage you to first test a traditional approach (like Conditional Random Fields) before going directly into state-of-the-art *Sesame-Street* character based models... 
+Scikit-learn models tend to be fast, but usually have lower accuracy than deep learning methods. However, for well defined problems with well defined features, they can provide very good results.
+Since deep learning models tend to be complex and slow, we encourage you to first test a traditional approach (like Conditional Random Fields) before going directly into state-of-the-art *Sesame-Street* character based models...
 When building such model, you would need to extend the [EntityRecognizer](presidio-analyzer/presidio_analyzer/entity_recognizer.py) class.
-   
+
 #### Apply custom logic
 
-In some cases, rule-based logic provides the best way of detecting entities. 
+In some cases, rule-based logic provides the best way of detecting entities.
 The Presidio EntityRecognizer API allows you to use spaCy extracted features like lemmas, part of speech, dependencies and more to create your logic. When building such model, you would need to extend the [EntityRecognizer](presidio-analyzer/presidio_analyzer/entity_recognizer.py) class.
 
 #### Deep learning based methods
 
-Deep learning methods offer excellent detection rates for NER. 
-They are however more complex to train, deploy and tend to be slower than traditional approaches. 
+Deep learning methods offer excellent detection rates for NER.
+They are however more complex to train, deploy and tend to be slower than traditional approaches.
 When contributing a DL based method, the best option would be to create a sidecar container which is isolated from the presidio-analyzer container. On the `presidio-analyzer` side, one would extend the [RemoteRecognizer](presidio-analyzer/presidio_analyzer/remote_recognizer.py) class and implement the network interface between `presidio-analyzer` and the endpoint of the model's container.
 
+## Language support
+When developing a new recognizer, a supported language should be specified ('en' by default if not specified). When a request for analysis is made it includes the language as argument. Only recognizers supporting that language will be used (if any). A recognizer can support a single language only.
```

Comment:
Let's recommend to adopt the ISO 639-1 language code list.
https://www.iso.org/standard/22109.html
https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes